src/ucp/tag/offload.h

```diff
@@ -66,6 +67,10 @@ void ucp_tag_offload_cancel(ucp_worker_t *worker, ucp_request_t *req, int force)
 
 int ucp_tag_offload_post(ucp_request_t *req, ucp_request_queue_t *req_queue);
 
+/* Returns 0 if tag offloading is disabled in the configuration,
+ * returns 1 otherwise */
```

Comment:
if you already document the function, why not to use doxygen style. 
Also, why not bool ?

Reply:
+1, bool it's C++ only AFAIK

Reply:
C99 has within #include <stdbool.h>

Reply:
Just was consistent, we use int when need bool in other places

Reply:
n ugni code we already make the assumption (c99) about bool
https://github.com/openucx/ucx/blob/51be945ed14d6a7fcbb855401a9a8e2fb518c17b/src/uct/ugni/base/ugni_types.h#L13

Reply:
in UCP/UCT we do not use that, so I'd not change it to bool here

Reply:
One of the two places would have to change.

Reply:
I'll update the comment. Or you meant something else?

Reply:
Please update comment. As for C99/bool - we would have to fix here or in ugni code.

Reply:
I'd stay with int. Then ugni needs to be modified in a separate PR as it is not related to this feature. @yosefe , what is your opinion?

Reply:
int

Reply:
please open an bug for @MattBBaker

Reply:
@shamisp, done - #2108